include/alpaka/offset/Traits.hpp

```diff
@@ -19,100 +20,168 @@ namespace alpaka
         //!
         //! If not specialized explicitly it returns 0.
         template<typename TIdx, typename TOffsets, typename TSfinae = void>
-        struct GetOffset
-        {
-            ALPAKA_NO_HOST_ACC_WARNING
-            ALPAKA_FN_HOST_ACC static auto getOffset(TOffsets const&) -> Idx<TOffsets>
-            {
-                return static_cast<Idx<TOffsets>>(0);
-            }
-        };
-
-        //! The x offset set trait.
-        template<typename TIdx, typename TOffsets, typename TOffset, typename TSfinae = void>
-        struct SetOffset;
+        struct [[deprecated("Specialize GetOffsets instead")]] GetOffset{
+            ALPAKA_NO_HOST_ACC_WARNING ALPAKA_FN_HOST_ACC static auto getOffset(TOffsets const&)
+                ->Idx<TOffsets>{return static_cast<Idx<TOffsets>>(0);
     } // namespace trait
+}; // namespace alpaka
```

Comment:
This comment and the comment above is wrong, please remove the comment.

Reply:
There is also another namespace squasing opportunity here :-)